src/BlockTypes/MiniCart.php

```diff
@@ -77,6 +107,26 @@ protected function enqueue_data( array $attributes = [] ) {
 		// Hydrate the following data depending on admin or frontend context.
 		if ( ! is_admin() && ! WC()->is_rest_api_request() ) {
 			$this->hydrate_from_api();
+
+			$label_info = $this->get_tax_label();
+
+			$label_including_tax               = $label_info['label_including_tax'];
+			$display_cart_prices_including_tax = $label_info['display_cart_prices_including_tax'];
+
+			$this->label_including_tax               = $label_including_tax;
+			$this->display_cart_prices_including_tax = $display_cart_prices_including_tax;
```

Comment:
Could this be compressed in two lines?

```suggestion
			$this->label_including_tax               = $label_info['label_including_tax'];
			$this->display_cart_prices_including_tax = $label_info['display_cart_prices_including_tax']
```